_includes/header.html

```diff
@@ -43,6 +43,6 @@ <h1>Just<span>Delete</span>Me</h1>
 
         <a class="button" href="https://github.com/jdm-contrib/jdm">Fork&nbsp;on&nbsp;GitHub</a>
 
-        <a class="button" href="https://twitter.com/intent/tweet?text=Just+Delete+Me+-+A+directory+of+direct+links+to+delete+your+account+from+web+services.+http%3A%2F%2Fjustdeletme.xyz">Tweet JDM</a>
+        <a class="button" href="https://twitter.com/intent/tweet?text=Just+Delete+Me+-+A+directory+of+direct+links+to+delete+your+account+from+web+services.+http%3A%2F%2Fjustdeleteme.xyz">Tweet JDM</a>
```

Comment:
Nice catch!

Reply:
It's the `e` of "delete": "justdeletme.xyz" vs "justdeletem**e**.xyz"

Reply:
Oh, I see. The URL was being removed from the diff view because of my adblock program. I see it fine after expanding the comment in the conversation view. 🙃

Reply:
ahhahah alright